v2/tools/generator/internal/codegen/pipeline/apply_kubernetes_resource_interface.go

```diff
@@ -14,43 +14,36 @@ import (
 	"github.com/Azure/azure-service-operator/v2/tools/generator/internal/interfaces"
 )
 
+const ApplyKubernetesResourceInterfaceStageID = "applyKubernetesResourceInterface"
+
 // ApplyKubernetesResourceInterface ensures that every Resource implements the KubernetesResource interface
 func ApplyKubernetesResourceInterface(idFactory astmodel.IdentifierFactory) Stage {
 	return MakeLegacyStage(
```

Comment:
Update this to use `MakeStage()` at the same time - we want to eventually remove `MakeLegacyStage()` completely.

Reply:
Done